### PR TITLE
fixes modifier hotkeys and adds alternative move intent toggle

### LINKED ISSUE #47563: Walk button is now a toggle
#47003 probably changed this. @Couls 

If you press alt you change movement mode completely, instead of just when held down.

This causes issues with alt tabbing, and alt clicking. Especially alt clicking bags which happen all the time.

### LINKED ISSUE #47570: OOC and say hotkeys now open two tabs, one has no cancel or X button
Hit T and you will be given two tabs. One is a "Say" tab with only an OK button and the other is a "say 'text'" tab with the normal OK, cancel, and x buttons.

OOC is the same way. Opens an "OOC" tab like above and an "ooc 'text'" tab. The new "Say" and "OOC" tabs also cant exist at the same time and will wait until one is closed for the other one to finally open.

Issue reported from  Round ID: 122903 (/tg/Station Sybil [ENGLISH] [US-WEST] [100% LAG FREE])

Reporting client version: 512.1488

### PR DESCRIPTION
**Pull request by Couls**
## About The Pull Request
Adds an alternative move intent toggle that doesn't restore original intent after pressing, it starts unbound also fixes #47563

should also fix #47570 but its hard to test because I still couldn't reproduce

## Changelog
:cl:
add: new button that truly toggles move intent
fix: hotkeys using Alt, Shift and Ctrl now work correctly
fix: pressing h will allow you to stop pulling now
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
will the save files automatically update when the testmerge is unmerged or do I need to bump it up to 27?

**Comment by actioninja** [comment]
save files will not automatically update, you're going to have to bump it again.

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -127,12 +127,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
...
 			S["facial_style_name"]	>> facial_hairstyle
 			
-	if(current_version < 26)
-		key_bindings = sanitize_islist(key_bindings, deepCopyList(GLOB.keybinding_list_by_key))
-		key_bindings["T"] = list(1 = "say")
-		key_bindings["M"] = list(1 = "me")
-		key_bindings["O"] = list(1 = "ooc")
-		key_bindings["L"] = list(1 = "looc")
+	if(current_version < 27)
+		key_bindings = deepCopyList(GLOB.keybinding_list_by_key)
```
Comment (Couls) [inline code/modules/client/preferences_savefile.dm]: this should reset everyone who is below version 27's keybindings to default, should clean up the weird ones added in version 26

Comment (actioninja) [reply code/modules/client/preferences_savefile.dm]: Best to nuke it from orbit, only way to be sure.

**Pull request merged**